consensus/types.go

```diff
@@ -110,33 +103,43 @@ type PrivateKey interface {
 	Public() PublicKey
 }
 
-// Signature is a cryptographic signature of a block.
-type Signature interface {
-	ToBytes
-	// Signer returns the ID of the replica that created the signature.
-	Signer() hotstuff.ID
-}
-
-// ThresholdSignature is a signature that is only valid when it contains the signatures of a quorum of replicas.
-type ThresholdSignature interface {
+// QuorumSignature is a signature that is only valid when it contains the signatures of a quorum of replicas.
+type QuorumSignature interface {
 	ToBytes
 	// Participants returns the IDs of replicas who participated in the threshold signature.
 	Participants() IDSet
 }
 
+// ThresholdSignature is a signature that is only valid when it contains the signatures of a quorum of replicas.
+//
+// Deprecated: renamed to QuorumSignature
+type ThresholdSignature = QuorumSignature
+
 // PartialCert is a signed block hash.
 type PartialCert struct {
-	signature Signature
+	// shortcut to the signer of the signature
```

Comment:
In general, I'm not sure the `consensus` package is the right place for these certificate types. I wonder if it wouldn't be better to place them in `crypto` or `crypto/certificates` or something along these lines.

Reply:
Marking this as resolved since I created an issue for the possible refactor of the module system.